tests/bluetooth/audio/mocks/include/bap_unicast_server_expects.h

```diff
@@ -155,6 +155,26 @@ do {
 				       func_name, "stream");))                                     \
 } while (0)
 
+#define expect_bt_bap_unicast_server_cb_release_called_twice(_streams)                             \
```

Comment:
This seems like it'll scale poorly :D 

Does it make sense to have a `expect_bt_bap_unicast_server_cb_release_called(_cnt, _streams)`? as a more generic approach? 
Not just for this, for for all the `expect_..._called_...`?